src/csv_reader/csv_reader.c

```
@@ -144,26 +165,20 @@
 // create a new CSV reader
 CSVReader CSVReader_New
 (
-    const char *file_name,  // URI to CSV
-    bool has_headers,       // first row is a header row
-    char delimiter          // column delimiter character
+	FILE *stream,      // CSV stream handle
+	bool has_headers,  // first row is a header row
+	char delimiter     // column delimiter character
 ) {
-	ASSERT(file_name != NULL);
+	ASSERT(stream != NULL);
```

cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)